ui/src/app/views/settings/component.tsx

```diff
@@ -46,7 +47,12 @@ export class SettingsReact extends React.Component<{}, SettingsState> {
   }
 
   componentDidMount() {
-    this.pollCluster();
+    userProfileStore.asObservable().subscribe((profileStore) => {
```

Comment:
Need to Observe here because `freeTierBillingProjectName` is not yet populated on initial load